doc/classes/Line2D.xml

```diff
@@ -58,8 +58,9 @@
 	</methods>
 	<members>
 		<member name="antialiased" type="bool" setter="set_antialiased" getter="get_antialiased" default="false">
-			If [code]true[/code], the line's border will be anti-aliased.
-			[b]Note:[/b] Line2D is not accelerated by batching when being anti-aliased.
+			If [code]true[/code], the line's border will be anti-aliased by drawing thin antialiased lines on the line's edges.
+			[b]Note:[/b] Line2D is not accelerated by batching if [member antialiased] is [code]true[/code].
+			[b]Note:[/b] Due to how it works, built-in antialiasing will not look correct for translucent lines and may not work on certain platforms. As a workaround, install the [url=https://github.com/godot-extended-libraries/godot-antialiased-line2d]Antialiased Line2D[/url] add-on then create an AntialiasedLine2D node. That node relies on a texture with custom mipmaps to perform antialiasing. 2D batching is also still supported with those antialiased lines.
```

Comment:
The note about 2D batching is only valid if 2D batching works when a texture is supplied. Is this the case? cc @lawnjelly

Reply:
Yes batching works when texture is applied, so the old comment is correct (if not great grammatically).

Your revised version of the comment, while being better English, may be less accurate. Anti-aliasing hint is not applied on mobile if I remember right (maybe the GLES_OVER_GL define?), so whether or not batching is used is not necessarily to do with whether the user has set this flag (which is a request), it is to do with whether the engine is trying to apply this flag.

Reply:
I reworded the descriptions to mention that antialiasing is only *attempted*.